core/templates/pages/contributor-dashboard-admin-page/contributor-dashboard-admin-page.module.ts

```diff
@@ -28,6 +28,23 @@ import { OppiaAngularRootComponent } from 'components/oppia-angular-root.compone
 import { ContributorDashboardAdminNavbarComponent } from './navbar/contributor-dashboard-admin-navbar.component';
 import { platformFeatureInitFactory, PlatformFeatureService } from 'services/platform-feature.service';
 import { HybridRouterModuleProvider } from 'hybrid-router-module-provider';
+import { ToastrModule } from 'ngx-toastr';
+
+// Config for ToastrModule (helps in flashing messages and alerts).
+const toastrConfig = {
+  allowHtml: false,
+  iconClasses: {
+    error: 'toast-error',
+    info: 'toast-info',
+    success: 'toast-success',
+    warning: 'toast-warning'
+  },
+  positionClass: 'toast-bottom-right',
+  messageClass: 'toast-message',
+  progressBar: false,
+  tapToDismiss: true,
+  titleClass: 'toast-title'
+};
```

Comment:
Can we store this config somewhere and import here and in core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.module.ts (and elsewhere if it's common)?

Reply:
done